index.js

```diff
@@ -23,12 +25,18 @@ function logger(winstonInstance) {
     if(this.status >=500) { logLevel = 'error'; }
     else if(this.status >=400) { logLevel = 'warn'; }
     else if(this.status >=100) { logLevel = 'info'; }
-
-    let msg = (chalk.gray(`${this.method} ${this.originalUrl}`) +
-               chalk[STATUS_COLORS[logLevel]](` ${this.status} `) +
-               chalk.gray(`${ms}ms`));
-
-    winstonInstance.log(logLevel, msg);
+    
+    const msgOne = `${this.method} ${this.originalUrl}`;
+    const msgTwo = ` ${this.status} `;
+    const msgThree = `${ms}ms`;
```

Comment:
We can name this variables with some more meaningful? e.g: `requestStatus`, `responseTime`